### PR TITLE
[prometheus-postgres-exporter] Add zanhsieh as maintainer

### PR DESCRIPTION
**Pull request by zanhsieh**
Signed-off-by: zanhsieh <zanhsieh@gmail.com>

I would like to volunteer as co-maintainer of prometheus-postgres-exporter as:

https://github.com/prometheus-community/helm-charts/issues/21

I currently am one of the maintainer of prometheus chart:

https://github.com/prometheus-community/helm-charts/blob/ed52a9d095291eb825010e0af89fb9bc365f054f/charts/prometheus/Chart.yaml#L17

### CODE REVIEW

File: charts/prometheus-postgres-exporter/Chart.yaml
Comment (scottrigby) [inline charts/prometheus-postgres-exporter/Chart.yaml:16]: You'd also need to add yourself to `/.github/CODEOWNERS`

**Review by scottrigby — CHANGES_REQUESTED** [review]
We'll also want to wait a bit for @gianrubio. Though I haven't heard replies from him for a while and he hasn't yet accepted the request to be part of this repo.

Related: we need to define the process (in PROCESSES doc) for what to do if chart maintainers are unreachable for some specified period of time.

File: charts/prometheus-postgres-exporter/Chart.yaml
Comment (zanhsieh) [reply charts/prometheus-postgres-exporter/Chart.yaml:16]: done

**Comment by zanhsieh** [comment]
@scottrigby 
When I change .github/CODEOWNERS, Github keeps complaining conflicting file .github/CODEOWNERS. What do I do wrong?

**Comment by torstenwalter** [comment]
Did you try merging main branch into yours? As alternative you could do `git checkout main .github/CODEOWNERS` make the change you want and commit that.

**Comment by scottrigby** [comment]
PS I heard from @gianrubio 🙂 so after fixing merge conflicts (as @torstenwalter said merge main), let's wait for Giancarlo's approval

File: .github/CODEOWNERS
Comment (scottrigby) [inline .github/CODEOWNERS:5]: This looks like it now accidentally reverts https://github.com/prometheus-community/helm-charts/pull/44. Can you check main again and if make sure only your change is in this PR?

**Review by scottrigby — CHANGES_REQUESTED** [review]

**Comment by gianrubio** [comment]
There's a conflict with the CODEOWNER file, @zanhsieh please rebase your branch and see if it help. Other than that I'm fine by adding you as a maintainer

**Comment by zanhsieh** [comment]
Close this since some rebase issue.

**Pull request closed without merging**
